diff --git a/tests/rn-versions/0.64/android/app/_BUCK b/tests/rn-versions/0.64/android/app/_BUCK
index 80c7e70..040a576 100644
--- a/tests/rn-versions/0.64/android/app/_BUCK
+++ b/tests/rn-versions/0.64/android/app/_BUCK
@@ -35,12 +35,12 @@ android_library(
 
 android_build_config(
     name = "build_config",
-    package = "com.testapp64",
+    package = "com.test64.app",
 )
 
 android_resource(
     name = "res",
-    package = "com.testapp64",
+    package = "com.test64.app",
     res = "src/main/res",
 )
 
diff --git a/tests/rn-versions/0.64/android/app/build.gradle b/tests/rn-versions/0.64/android/app/build.gradle
index 79e84a7..9701ed8 100644
--- a/tests/rn-versions/0.64/android/app/build.gradle
+++ b/tests/rn-versions/0.64/android/app/build.gradle
@@ -131,7 +131,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.testapp64"
+        applicationId "com.test64.app"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
diff --git a/tests/rn-versions/0.64/android/app/src/debug/java/com/testapp64/ReactNativeFlipper.java b/tests/rn-versions/0.64/android/app/src/debug/java/com/test64/app/ReactNativeFlipper.java
similarity index 99%
rename from tests/rn-versions/0.64/android/app/src/debug/java/com/testapp64/ReactNativeFlipper.java
rename to tests/rn-versions/0.64/android/app/src/debug/java/com/test64/app/ReactNativeFlipper.java
index 6ee532d..60d5b55 100644
--- a/tests/rn-versions/0.64/android/app/src/debug/java/com/testapp64/ReactNativeFlipper.java
+++ b/tests/rn-versions/0.64/android/app/src/debug/java/com/test64/app/ReactNativeFlipper.java
@@ -4,7 +4,7 @@
  * <p>This source code is licensed under the MIT license found in the LICENSE file in the root
  * directory of this source tree.
  */
-package com.testapp64;
+package com.test64.app;
 
 import android.content.Context;
 import com.facebook.flipper.android.AndroidFlipperClient;
diff --git a/tests/rn-versions/0.64/android/app/src/main/AndroidManifest.xml b/tests/rn-versions/0.64/android/app/src/main/AndroidManifest.xml
index 4745069..26afdde 100644
--- a/tests/rn-versions/0.64/android/app/src/main/AndroidManifest.xml
+++ b/tests/rn-versions/0.64/android/app/src/main/AndroidManifest.xml
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.testapp64">
+  package="com.test64.app">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
diff --git a/tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainActivity.java b/tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainActivity.java
similarity index 93%
rename from tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainActivity.java
rename to tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainActivity.java
index a2f226f..003db2a 100644
--- a/tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainActivity.java
+++ b/tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainActivity.java
@@ -1,4 +1,4 @@
-package com.testapp64;
+package com.test64.app;
 
 import com.facebook.react.ReactActivity;
 
diff --git a/tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainApplication.java b/tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainApplication.java
similarity index 96%
rename from tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainApplication.java
rename to tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainApplication.java
index 7ffdb00..7ce6787 100644
--- a/tests/rn-versions/0.64/android/app/src/main/java/com/testapp64/MainApplication.java
+++ b/tests/rn-versions/0.64/android/app/src/main/java/com/test64/app/MainApplication.java
@@ -1,4 +1,4 @@
-package com.testapp64;
+package com.test64.app;
 
 import android.app.Application;
 import android.content.Context;
@@ -62,7 +62,7 @@ public class MainApplication extends Application implements ReactApplication {
          We use reflection here to pick up the class that initializes Flipper,
         since Flipper library is not available in release mode
         */
-        Class<?> aClass = Class.forName("com.testapp64.ReactNativeFlipper");
+        Class<?> aClass = Class.forName("com.test64.app.ReactNativeFlipper");
         aClass
             .getMethod("initializeFlipper", Context.class, ReactInstanceManager.class)
             .invoke(null, context, reactInstanceManager);
